internal/app/frontend/frontend_service_test.go

```diff
@@ -32,48 +32,67 @@ import (
 )
 
 func TestDoCreateTickets(t *testing.T) {
-	store, closer := statestoreTesting.NewStoreServiceForTesting(t, viper.New())
+	cfg := viper.New()
+	cfg.Set("playerIndices", []string{"test-property"})
+	store, closer := statestoreTesting.NewStoreServiceForTesting(t, cfg)
 	defer closer()
 
 	normalCtx := context.Background()
 	cancelledCtx, cancel := context.WithCancel(context.Background())
 	cancel()
 
-	testTicket := &pb.Ticket{
+	numTicket := &pb.Ticket{
 		Properties: &structpb.Struct{
 			Fields: map[string]*structpb.Value{
 				"test-property": {Kind: &structpb.Value_NumberValue{NumberValue: 1}},
 			},
 		},
 	}
 
+	stringTicket := &pb.Ticket{
+		Properties: &structpb.Struct{
+			Fields: map[string]*structpb.Value{
+				"test-property": {Kind: &structpb.Value_StringValue{StringValue: "string"}},
+			},
+		},
+	}
+
 	tests := []struct {
 		description string
+		ticket      *pb.Ticket
 		ctx         context.Context
 		wantErr     bool
 	}{
+		{
+			description: "expect precondition error since open-match does not support properties other than number",
+			ctx:         normalCtx,
+			ticket:      stringTicket,
```

Comment:
simply initialize the ticket objects here instead of having them in named variables just used here. Your description already covers the condition.